components/Projects/FiltersBar/FiltersWindow.tsx

```diff
@@ -0,0 +1,79 @@
+import Radiobox from '@/components/utils/Radiobox';
+import React, { Ref } from 'react';
+import { useTranslations } from 'next-intl';
+import FilterTagBtn from './FilterTagBtn';
+import { ProjectFilter } from '@/types';
+
+interface FiltersWindow {
+  filterRef: Ref<HTMLDivElement>;
+  sortOptions: string[];
+  selectedSortOption: any;
+  handleCategoryOptionSelection: any;
+  filters: ProjectFilter[];
+  handleFilterOptionChange: any;
+}
+
```

Comment:
Please change the name to `FiltersWindowProps` like we do in the project for components props. 

Also don't use `any` , change your interface to have defined values.